README.md

```diff
@@ -81,66 +53,11 @@ Once installed, you can run a temporary deployment in dev mode (all data is in-m
 coder server --dev
 ```
 
-Use `coder --help` to get a complete list of flags and environment variables.
-
-## Creating your first template and workspace
-
-In a new terminal window, run the following to copy a sample template:
-
-```bash
-coder templates init
-```
-
-Follow the CLI instructions to modify and create the template specific for your
-usage (e.g., a template to **Develop in Linux on Google Cloud**).
-
-Create a workspace using your template:
-
-```bash
-coder create --template="yourTemplate" <workspaceName>
-```
-
-Connect to your workspace via SSH:
-
-```bash
-coder ssh <workspaceName>
-```
-
-## Modifying templates
-
-You can edit the Terraform template using a sample template:
-
-```sh
-coder templates init
-cd gcp-linux/
-vim main.tf
-coder templates update gcp-linux
-```
+Use `coder --help` to get a complete list of flags and environment variables. For a full walkthrough, follow [this guide](./docs/walkthrough.md).
```

Comment:
Suggest changing to Use our [quickstart guide](./docs/walkthrough.md) for a full walkthrough

Reply:
I like that! changed